Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:14.04
+
+ADD ./WcaOnRails /app/WcaOnRails
```

Comment:
> Having this line at the top causes a rebuild of the entire image every time you change a file which is rather annoying when developing.

I think this is not true, to my understanding ADD basically mounts the folder so when you do changes you don't have to rebuild.